### DISCUSSION TITLE
Cannot understand how import works

### DISCUSSION THREAD

**Discussion by Serhioromano** [eclipse-langium/langium#794] [Q&A]
I have a base file where I have basic structure and I want some other rules to be in the other file. I created another file and add `import "./others"`  That is ok. But in that others file I want to use same terminals and it is highlighted as not imported.

If I create separate file with terminals and import it to my main file and others, it is still not working, not imported.

Does that mean that in every grammar file I have to create terminals and if it required same terminals?

**Comment by spoenemann** [comment]
It should be possible to import terminals defined in a separate file. Could you give an example?

**Reply by Serhioromano** [reply]
2 files

terminal.langium
```
terminal ID: /[a-z]+/;
```

grammar.lamgium
```
grammar Test
import "./terminal.langium"

entry Document:
    'Name' name=ID
```

**Reply by spoenemann** [reply]
It works with the current main branch. Maybe it's a bug that has been fixed in the meantime. The next release is planned for December 15.
